core/templates/components/button-directives/primary-link-button.component.html

```diff
@@ -0,0 +1,15 @@
+<a *ngIf="!isButton" class="oppia-primary-link-button"
```

Comment:
Could you please do a pass to check accessibility for this button/link? Some best practices are here and there might be other resources: https://www.deque.com/blog/accessible-aria-buttons/

Reply:
Done